DigitalLearningSolutions.Web.IntegrationTests/DigitalLearningSolutions.Web.IntegrationTests.csproj

```diff
@@ -6,6 +6,10 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <AdditionalFiles Remove="C:\Users\Sherif.Olaboye\Source\Repos\DLSV2\StyleCop.json" />
```

Comment:
I'm not sure why this has  been added. It seems to reference a path on your local machine.

Reply:
This is already remove from the project , I need to remove this file cause am not sure if it is needed, even when I pull from master branch to compare the file was there for the project to build and run the is need to be remove